### PR TITLE
`Development`: Add documentation for learning analytics

### PR DESCRIPTION
**Pull request by maximiliansoelch**
#### General

- [x] Language: I followed the [guidelines for inclusive, diversity-sensitive, and appreciative language](https://docs.artemis.ase.in.tum.de/dev/guidelines/language-guidelines/).
- [x] I chose a title conforming to the [naming conventions for pull requests](https://artemis-platform.readthedocs.io/en/latest/dev/guidelines/development-process.html#naming-conventions-for-github-pull-requests).

### Motivation and Context

Documentation about Artemis' learning analytics features are missing.

### Description

Add documentation about all available learning analytics for students and instructors.

### Steps for Testing

1. Look at the new documentation about learning analytics: https://artemis-platform--5364.org.readthedocs.build/en/5364/user/learning-analytics/

#### Code Review
- [x] Review 1
- [x] Review 2

### CODE REVIEW

**Review by ge65cer — APPROVED** [review]
Sounds good to me. 🙂

**Review by MetinLamby — DISMISSED** [review]
really nice documentation! Concise and clear!

File: docs/user/learning-analytics.rst [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+
+Overall Points
+^^^^^^^^^^^^^^
+A donut diagram shows students their overall achieved points in the course.
+
+|students-overall-points|
+
+Performance in Exercises
+^^^^^^^^^^^^^^^^^^^^^^^^
+Students can compare their scores to the best, overall score as well as the average score for each individual exercise with the help of a line chart.
```
Comment (jschmautzer) [inline docs/user/learning-analytics.rst:29]:
```suggestion
Students can compare their scores to the best overall score as well as the average score for each individual exercise with the help of a line chart.
```

**Review by jschmautzer — COMMENTED** [review]
Regarding the "Exercise Statistics" Section: Maybe add a little description how someone can get to this exercise statistics overview.

The rest looks understandable :)

File: README.md [hunk 2]
```diff
@@ -38,7 +38,7 @@
 12. **Lectures**: Instructors can upload lecture slides, divide lectures into units, integrate video streams, lecture recordings, and exercises into lectures, and define learning goals.
 13. **Integrated markdown editor**: Instructors define problem statements using Markdown to easily format the text and integrate images and links.
 14. **Plagiarism checks**: Artemis integrates plagiarism checks for programming exercises (based on [JPlag](https://github.com/jplag/JPlag)), text exercises, and modeling exercises. It allows notifying students about identified plagiarism. Students can review and comment on the allegation. 
-15. **Learning analytics**: Artemis integrated different statistics for students to compare themselves to the course average. It allows instructors to evaluate the average student performance based on exercises and learning goals.
+15. **[Learning analytics]((https://docs.artemis.ase.in.tum.de/user/learning-analytics/))**: Artemis integrated different statistics for students to compare themselves to the course average. It allows instructors to evaluate the average student performance based on exercises and learning goals.
```
Comment (bassner) [inline README.md:41]: Why the doubled brackets? :)
```suggestion
15. **[Learning analytics](https://docs.artemis.ase.in.tum.de/user/learning-analytics/)**: Artemis integrated different statistics for students to compare themselves to the course average. It allows instructors to evaluate the average student performance based on exercises and learning goals.
```

**Review by bassner — CHANGES_REQUESTED** [review]

**Review by Marvin-Vitcu — APPROVED** [review]
Tested on TS1. The link is there and works as intended.

**Review by bassner — DISMISSED** [review]

**Review by Mikael31 — DISMISSED** [review]
Good documentation and understandable for everyone

**Review by jschmautzer — DISMISSED** [review]
documentation looks godd :)

**Review by luisa-we — APPROVED** [review]
Good documentation!
Tested on TS1, the link is present and works.

**Review by sleiss — APPROVED** [review]
Documentation looks good :+1:

**Review by krusche — CHANGES_REQUESTED** [review]
Good starting point, could you mention the learning goal statistics for students as well?

**Review by babyygemperor — APPROVED** [review]
Good documentation. Looks good 👍🏼

**Review by luisa-we — APPROVED** [review]
Documentation looks good to me!

**Review by rhubinger — APPROVED** [review]
Read through it, looks good!

**Review by bassner — APPROVED** [review]
LGTM!

**Review by ge65cer — APPROVED** [review]
Re-approve

**Review by Mikael31 — APPROVED** [review]
reviewed. looks good

**Pull request merged**